### ISSUE TITLE
Front End Components - home page

### ISSUE THREAD

**Issue by rb50** [Johanna-hub/BrixtonLocal#10]
Create Front End React components for the landing page (please see the figma design):

- [x] Navbar
- [ ] Main Content container to include:
   - [x] Landing page to include:
        - [x] Brixton local.life header
        - [x] Horizontal scroll categories
   - [ ] Top 5-10 tags 
   - [x] Card listing to include:
        - [x]  Photo (if provided) or standard illustration (if no photo)
        - [x] Category
        - [x] Whether delivering or collecting
        - [x] Entry name
        - [x] Entry tags
    - [ ] All tags
    - [ ] Footer

**Comment by Johanna-hub** [comment]
I would really love the individual business listings to be in a random order when you arrive, and different every time you come back.  Then putting them into alphabetical is an option if you want.

Is that mad?  it's just i feel like the A and B companies get so much more exposure!!!

**Comment by Johanna-hub** [comment]
Obviously if you click on a tag or category it would be alphabetical because that is much smaller.

**Comment by rb50** [comment]
Yes I actually thought about this too! I think we can shuffle the array of listings from google sheets randomly using Math.random somehow.
